constellation-app/constellation
# Issue#19 corrup star file handling

### Description of the Change

Fixed bugs found in existing fixes for corrupt files - in particular, incorrect file sperator constant and code to ensure zero sized fiels aren't copied.

### Alternate Designs

nil

### Why Should This Be In Core?

bug fix

### Benefits

prevent 0 sized files being created as backups.
prevent issues where file copy may fail due to seperator.

### Possible Drawbacks

nil
### Verification Process

1. Open Constellation
2. Create a new graph
3. Draw a vertex on graph
4. Select File-Save and save the graph. Confrim no .bak file is created
5. Draw another vertex on the graph
6. Select File-Save and save the graph. confirm a .bak file exists that is non zero size

### Applicable Issues

https://github.com/constellation-app/constellation/issues/19
